health/notifications/smstools3/README.md

```diff
@@ -4,6 +4,12 @@ The [SMS Server Tools 3](http://smstools3.kekekasvi.com/) is a SMS Gateway softw
 
 To have netdata send notifications via SMS Server Tools 3, you'll first need to [install](http://smstools3.kekekasvi.com/index.php?p=compiling) and [configure](http://smstools3.kekekasvi.com/index.php?p=configure) smsd.
 
+Ensure that the user `netdata` can execute `smssend`. Any user executing `smssend` needs to:
+- Have write permissions to `/tmp` and `/var/spool/sms/outgoing`
+- Be a member of group `smsd`
+
+To ensure that the steps above are successful, just `su netdata` and execute `sendsms phone message`.
```

Comment:
`sendsms` doesn't match up here with what is listed above (`smssend`).

Reply:
Thanks, fixed. You have no idea how many times I've written it the wrong way, and I'm not even dyslexic :)